sid/build/system.sh

```diff
@@ -110,6 +110,8 @@ apt install --no-install-recommends -y \
 	tumbler-common \
 	tumbler-plugins-extra
 
+rm -rfv /usr/share/backgrounds/*
+
```

Comment:
I'm confused sid and bookworm remove the backgrounds but nothign else? All other distros look to copy over a background image to tmp, remove the usr shared backgrounds dir completely, then mv the tmp background into it?

Reply:
Totally understandable and I was confused at first too. Typically, a lot of these distros ship pre-themed wallpapers. i.e. `rocky-wallpapers`. In the case of Debian, they don't. So I have to manually download the wallpaper and ship it with git. https://github.com/juno-fx/Helios/pull/25/files#diff-c7682c6ecaf336b60ba16fb2e351061146b575f70257c94a838e2e6853bd8964

And since I copy the root location, I don't need to actually move or rename any files since I just drop it into the location manually in the repo. 

I am essentially doing the same thing, but these are just setup a little different since debian doesn't distribute the same way as the others.